go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/operator-framework/helm-operator-plugins
 
-go 1.22.5
+go 1.22.1
```

Comment:
I guess I have to read up on this, but why `1.22.1` vs `1.22`? 🤔

Reply:
So, if I do 1.22 and go mod tidy:

```
go 1.22.0

toolchain go1.23.3
```

I initially started with 1.22.0, but the lint tool in bingo needs 1.22.1 :shrug: I get that's not a great reason. And probably something we want to think about going forward. But, I think it's good enough for now to ensure we don't block folks downstream from here.

Reply:
But, my understanding is, toolchain allows people on older Go versions to still vendor in the project. The Go compiler would then download the required toolchain, etc.

Reply:
How mod tidy resolves the go version and toolchain version is not entirely clear to me - should probably read up on that as well XD